backend/btrixcloud/crawls.py

```diff
@@ -162,6 +164,11 @@ class ListCrawlOut(BaseMongoModel):
 
     stopping: Optional[bool] = False
 
+    collections: Optional[List[UUID4]] = []
+
+    files: Optional[List[CrawlFile]] = []
```

Comment:
One issue with adding `files` and `resources` here is that they will be included on all the list responses, as this is the ListCrawlOut model. It looks like both are only used internally, so maybe we just add a ListCrawlOutWithResources model which extends ListCrawlOut and adds these two fields, and use that when `resources=True`?
Or, could add a `cls=ListCrawlOut` and add resources if `cls=ListCrawlOutWithResources` is passed in?

Reply:
I was working around that by manually setting files and resources to `None` in the endpoint before returning, but your idea of extending in a new class is better.

Reply:
Updated